lib/la-arena/src/map.rs

```diff
@@ -252,6 +252,7 @@ where
 {
     /// Ensures a value is in the entry by inserting the default value if empty, and returns a mutable reference
     /// to the value in the entry.
+    #[allow(clippy::unwrap_or_default)]
```

Comment:
Is this a false positive of the lint? (would be good to add a comment in that case for those)

Reply:
Added a comment